com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/RemoveNewApplicationsSuffixStep.java

```diff
@@ -37,16 +37,16 @@ protected StepPhase executeStep(ExecutionWrapper execution) {
             client.rename(appName, newName);
         }
 
-        String mtaId = (String) execution.getContext()
-                                         .getVariable(Constants.PARAM_MTA_ID);
-        updateConfigurationSubscribers(appsToProcess, mtaId);
+        updateConfigurationSubscribers(appsToProcess, execution.getContext());
 
         return StepPhase.DONE;
     }
 
-    private void updateConfigurationSubscribers(List<String> appsToProcess, String mtaId) {
+    private void updateConfigurationSubscribers(List<String> appsToProcess, DelegateExecution context) {
```

Comment:
I'd prefer passing the space GUID in addition to the MTA ID, instead of passing the context, since the context is just a map of variables. It's like passing everything to this method **and allowing it to change it**.

Reply:
Done